Sources/Vapor/HTTP/HTTPMethod+String.swift

```diff
@@ -5,6 +5,7 @@ extension HTTPMethod {
     public var string: String {
         switch self {
         case .GET: return "GET"
+        case .RAW(let value): return value
         default: return "\(self)"
         }
```

Comment:
The correct fix to this is to use the `rawValue` accessor provided by NIO (ostensibly, this `string` accessor shouldn't exist at all - it's only here because NIO didn't add `rawValue` until a few months after Tanner added this and he never went back to clean it up - but it's public API in Vapor so we can't just remove it, unfortunately).
```suggestion
        self.rawValue
```